### PR TITLE
[bitnami/wavefront-adapter-for-istio] Add chart

### PR DESCRIPTION
**Pull request by javsalgar**
**Description of the change**

This PR creates a first version of the Wavefront Adapter for Istio chart. This is based on the upstreamhttps://github.com/wavefrontHQ/helm/blob/master/wavefront-adapter-for-istio/ chart. It is  comprised of a Deployment, a Service and several Istio objects. It includes the Wavefront subchart, but we only use the proxy component (from upstream, we can see that only that is supported, so we disable the rest of the components by default). It contains the all the Bitnami standard specs and readmenator is used for rendering the tables.

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/wavefront-adapter-for-istio/templates/istio/adapter.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
...BhIDdAgPChAKCQQAAwIDAAIBARIDdBAXChAKCQQAAwIDAAIBAxIDdBobCj8KCAQAAwIDAAMAEgR4Bn4HGi0gRGVzY3JpYmVzIGFuIGV4cG9uZW50aWFsbHkgZGVjYXlpbmcgc2FtcGxlLgoKEAoJBAADAgMAAwABEgN4DhYKKAoKBAADAgMAAwACABIDegghGhUgVGhlIHJlc2Vydm9pciBzaXplLgoKEwoLBAADAgMAAwACAAQSBHoIeBgKEgoLBAADAgMAAwACAAUSA3oIDQoSCgsEAAMCAwADAAIAARIDeg4cChIKCwQAAwIDAAMAAgADEgN6HyAKHwoKBAADAgMAAwACARIDfQgZGgwgVGhlIGFscGhhLgoKEwoLBAADAgMAAwACAQQSBH0IeiEKEgoLBAADAgMAAwACAQUSA30IDgoSCgsEAAMCAwADAAIBARIDfQ8UChIKCwQAAwIDAAMAAgEDEgN9FxgKMQoIBAADAgMAAwESBoEBBoQBBxodIERlc2NyaWJlcyBhIHVuaWZvcm0gc2FtcGxlLgoKEQoJBAADAgMAAwEBEgSBAQ4VCikKCgQAAwIDAAMBAgASBIMBCCEaFSBUaGUgcmVzZXJ2b2lyIHNpemUuCgoVCgsEAAMCAwADAQIABBIGgwEIgQEXChMKCwQAAwIDAAMBAgAFEgSDAQgNChMKCwQAAwIDAAMBAgABEgSDAQ4cChMKCwQAAwIDAAMBAgADEgSDAR8gCjYKBAQAAwMSBokBAowBAxomIERlc2NyaWJlcyB0aGUgbG9nZ2luZyBjb25maWd1cmF0aW9uLgoKDQoFBAADAwESBIkBCg4KVAoGBAADAwIAEgSLAQQVGkQgVGhlIGxvZyBsZXZlbCAob25lIG9mIGVycm9yLCB3YXJuLCBpbmZvLCBkZWJ1Zywgb3Igbm9uZSkuIEV4OiBpbmZvCgoRCgcEAAMDAgAEEgaLAQSJARAKDwoHBAADAwIABRIEiwEECgoPCgcEAAMDAgABEgSLAQsQCg8KBwQAAwMCAAMSBIsBExQKJgoEBAACBhIEjwECEBoYIFRoZSBsb2cgY29uZmlndXJhdGlvbi4KCg8KBQQAAgYEEgaPAQKMAQMKDQoFBAACBgYSBI8BAgYKDQoFBAACBgESBI8BBwsKDQoFBAACBgMSBI8BDg9iBnByb3RvMw==
```
Comment (alemorcuq) [inline bitnami/wavefront-adapter-for-istio/templates/istio/adapter.yaml:22]: I imagine this is the output from the command in line 3?

File: bitnami/wavefront-adapter-for-istio/templates/istio/handler.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+apiVersion: {{ include "wfafi.istio.apiVersion" . }}
+kind: handler
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: wavefront-adapter-for-istio
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ include "common.names.fullname" . }}-wavefront-handler
+  namespace: {{ .Values.istio.namespace }}
```
Comment (alemorcuq) [inline bitnami/wavefront-adapter-for-istio/templates/istio/handler.yaml:11]: Some templates under `templates/istio` use `.Release.Namespace` and others `.Values.istio.namespace`. Is that correct?

**Review by alemorcuq — CHANGES_REQUESTED** [review]

File: bitnami/wavefront-adapter-for-istio/ci/values-external.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+wavefront:
+  enabled: false
+
+externalProxy:
+  host: test-proxy
+  port: 1234
```
[the remark this answers is not in the document]
Comment (javsalgar) [reply bitnami/wavefront-adapter-for-istio/ci/values-external.yaml:6]: This is for the hooks.

File: bitnami/wavefront-adapter-for-istio/templates/istio/adapter.yaml (hunk 1 above)
Comment (javsalgar) [reply bitnami/wavefront-adapter-for-istio/templates/istio/adapter.yaml:22]: Yes

File: bitnami/wavefront-adapter-for-istio/templates/istio/handler.yaml (hunk 2 above)
Comment (javsalgar) [reply bitnami/wavefront-adapter-for-istio/templates/istio/handler.yaml:11]: Yes, some of the objects need to be deployed in the istio namespace

**Review by alemorcuq — APPROVED** [review]

File: bitnami/wavefront-adapter-for-istio/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 1.0.3
+dependencies:
+  - condition: wavefront.enabled
+    name: wavefront
+    repository: https://charts.bitnami.com/bitnami
+    version: 1.x.x
```
Comment (carrodher) [inline bitnami/wavefront-adapter-for-istio/Chart.yaml:9]: There is a new major version
```suggestion
    version: 2.x.x
```

**Pull request merged**